Horse-racing Duals/tests/index.test.js

```diff
@@ -1,21 +1,32 @@
-const testSubject = require('../src/index');
-const fs = require('fs');
-const path = require("path");
+describe("Index.js function",() => {
+  const testSubject = require('../src/index');
+  const fs = require('fs');
+  const path = require("path");
 
-test('Simple case. Smallest difference is 1', () => {
-  const input = fs.readFileSync(path.join(__dirname, "simpleCase.txt"), 'utf-8');
-  const output = testSubject(input);
-	expect(output).toBe(1);
-});
+  beforeAll(() => {
+    console.log = jest.fn();
```

Comment:
I'm not familiar with Jest, however you need to make sure, that after this test suite is done, console.log continues to work as expected. You can make sure by reading documentation about jest.resetAllMocks, and what it does exactly